decisions/0002-record-architecture-decisions.md

```diff
@@ -0,0 +1,49 @@
+# 2. Communicating NIST's OSCAL Events to the Public
+
+Date: 2023-02-03
+
+## Status
+
+Feedback Requested
+
+## Context
+
+The NIST OSCAL Team has necessary touchpoints with the community, among them structured and unstructured meetings with community stakeholders. Many of these meetings are events occuring at set frequencies, while others do not. In all cases, the community, past and present, has expressed the need for invitations to events that are easily accessible with the most current information possible.
+
+## Decision
```

Comment:
I do not think that people (a.k.a. every member of the community) will be willing to open this calendar and merge it into their client. I thought we will use this calendar to invite oscal-dev@list.nist.gov to our event. The advantage of using this calendar would be the fact that any team member would be able to manage the invitations.

Reply:
I address it in a subsequent change https://github.com/usnistgov/OSCAL/pull/1638/commits/cd973e7a0362402c6f708d313ace8701b139bfbf.